Form/Type/DocumentType.php

```diff
@@ -66,16 +66,22 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
             return $registry->getManager($manager);
         };
 
-        $resolver->setNormalizers(array(
-            'em' => $normalizer,
-        ));
+        $resolver->setNormalizer('em', $normalizer);
     }
 
     /**
-     * @see Symfony\Component\Form\FormTypeInterface::getName()
+     * {@inheritdoc}
      */
     public function getName()
     {
+        return $this->getBlockPrefix();
```

Comment:
I could swear I saw it in the FormTypeInterface just a minute ago. You're right again. Thanks!